components/server/src/api/organization-service-api.ts

```diff
@@ -344,7 +344,7 @@ export class OrganizationServiceAPI implements ServiceImpl<typeof OrganizationSe
                     "onboardingSettings can only be set on enterprise installations",
                 );
             }
-            if (req.onboardingSettings.internalLink?.length ?? 0 > 255) {
+            if ((req.onboardingSettings.internalLink?.length ?? 0) > 255) {
```

Comment:
This is a fix for a regression from https://github.com/gitpod-io/gitpod/pull/20508. Operator precedence is hard :P